### DISCUSSION TITLE
Add UNLOAD_FILAMENT_AFTER_PRINT macro

### DISCUSSION THREAD

**Discussion by HitLuca** [jschuh/klipper-macros#279] [Ideas]
I was thinking about an interesting addition for this project, a macro to call UNLOAD_FILAMENT after the print is finished. Would this be feasible? I'm on an old-ish version of this project, so apologies if it has already been implemented. Let me know!

**Comment by stablestud** [comment]
You can add UNLOAD_FILAMENT to the end gcode macro in your slicer
